README.md

```diff
@@ -359,25 +412,29 @@ responsiblity of the end user to upgrade StackStorm according to the
 In a future release a Puppet task may be included to perform these update 
 on demand using [bolt](https://github.com/puppetlabs/bolt).
 
-## Known Limitations
-
 ### Ubuntu 16.04
 
 In StackStorm `< 2.4.0` there is a known bug [#3290](https://github.com/StackStorm/st2/issues/3290) 
 where the first puppet run fails to install the `st2` pack. Simply invoking puppet
 a second time will produce a fully functional installation with the `st2` pack 
 installed. This was an upstream bug and has been fixed in StackStorm version `2.4.0`.
```

Comment:
With latest st2 `3.1.0` and `2.4.0` released in 2017, I think it's OK to remove this "known limitation".